File: mentees/JayKawa/algorithms/searching/linear_search.c
#include<stdio.h>

int main()
{
    int a[100];
    int n,f,found;

    printf("Enter the number of elements you want :\n");
    scanf("%d",&n);

    for(int i=0;i<n;i++)
    {
        printf("Enter the %d element :\n",i+1);
        scanf("%d",&a[i]);
    }

    printf("Enter the element you to get :\n");
    scanf("%d",&f);

    found =0;

    for(int i =0;i<n;i++)
    {
        if(f==a[i])
        {
            printf("Element %d found at %d position.\n",f,i+1);
            break;
        }
    }

    if(!found)
    {
        printf("Element not found.");
    }

    return 0;
}

File: mentees/JayKawa/algorithms/searching/Linear_search.java
import java.util.*;

public class Linear_search {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in); 
        int n, f, found;

        System.out.println("Enter the number of elements you want:");
        n = sc.nextInt();
        int a[] = new int[n];

        for (int i = 0; i < n; i++) 
        {
            System.out.println("Enter the " + (i + 1) + " element:");
            a[i] = sc.nextInt();
        }

        System.out.println("Enter the element you want to find:");
        f = sc.nextInt();

        found = 0; 

        for (int i = 0; i < n; i++) 
        {
            if (a[i] == f) 
            {
                System.out.println("Element " + f + " found at position " + (i + 1) + ".");
                found = 1;  
                break; 
            }
        }

        if (found == 0) 
        {

            System.out.println("Element not found.");
        }

        sc.close();  
    }
}

